.github/workflows/pr.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
```

Comment:
I'm not going to look at any of the lein or CI changes. If anything, I'd accept a separate PR to move from lein to deps.edn.